### PR TITLE
The Great Species Food Rebalance, Part One: Nutriment, Chemical Recipes, Cargo Orders, Baking Times, Oh My!

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
A comprehensive rebalance of species food, based on feedback I've been given over the past 6 months.

When the two species food PRs launched, they were both rather undertuned in terms of nutritional value. This PR bumps the numbers up, with many foods receiving buffs by as much as 2x. This sounds like a lot, but this really brings the food mostly in line with existing food from other categories (yes, it was that undertuned at launch).

Recipes for the three major chems are included: Yoghurt (10u Cream, 2u Virus Food), Cornmeal (Grind Corn), and Quality Oil (1u Quality Oil, 2u Cooking Oil). The recipes for yoghurt and cornmeal are, I think, fairly self explanatory (kudos given to https://github.com/tgstation/tgstation/pull/66906), but the quality oil one may require some further explanation. Generally, I want quality oil to be a somewhat restricted material, hence why it was originally set up as an order-console exclusive. This has had a _lot_ of feedback, very little of it happy, so I've opted to relent in making it craftable, but still requiring that initial purchase. This means you can stretch your supply indefinitely on a single bottle.

Baking times are another area of contention. In some cases, it was necessary to sit for around 30 seconds waiting on the oven, which forms a pretty substantial speedbump in chef gameplay. While I don't want to eliminate it entirely, this PR reduces the oven bake times by around 1/2th. Most items will now bake in 15-25 seconds, rather than 25-45 seconds, which should make for a substantial reduction, although this can again be revised if it's still too long (or if it's managed to become too short, somehow).

Finally, a new selection of cargo items is here, aiming to make lizard and mothic food more accessible. Each crate includes a few boxes with a selection each of supplies from their respective cultures, for the low(ish) price of 300 credits (so as to not make them beyond the reach of the chef and his paycheck).

With these changes together, species food should (hopefully) be less frustrating to make and more rewarding once it's out. In a future PR, I'd like to take a look at the crafting recipes, including (hopefully) a system for getting more than one item/craft (which is non-trivial and that's why it's not in this PR).

## Why It's Good For The Game
Species food has had good feedback in flavour terms, but the dominant constructive feedback for both is that they're too difficult to make. With this PR, it should be easier to access this new content, and that helps with the flavour of our universe.

## Changelog

:cl:
add: A few new crates have made their way to cargo: buy yourself a Lizard or Moth food crate today!
add: Recipes for Yoghurt (10u cream, 2u virus food), Cornmeal (grind corn), and Quality Oil (1u quality oil, 2u cooking oil) have been added. Bon appetit!
balance: Species food (lizard and moth food) have received a sweep of balance changes: they're now more filling and a bit easier to access.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/food_mixtures.dm [hunk 1]
```diff
@@ -249,3 +249,15 @@
...
+	required_reagents = list(/datum/reagent/consumable/cream = 10, /datum/reagent/consumable/virus_food = 2)
+	results = list(/datum/reagent/consumable/yoghurt = 10)
+	mix_message = "The mixture thickens into yoghurt."
+	reaction_flags = REACTION_INSTANT
+
+/datum/chemical_reaction/food/quality_oil_upconvert
+	required_reagents = list(/datum/reagent/consumable/quality_oil = 1, /datum/reagent/consumable/cooking_oil = 2)
+	results = list(/datum/reagent/consumable/quality_oil = 2)
+	mix_message = "The cooking oil dilutes the quality oil- how delightfully devilish..."
+	reaction_flags = REACTION_INSTANT
```
Comment (carshalash) [inline code/modules/food_and_drinks/recipes/food_mixtures.dm:263]: It makes sense and I know people do it but it still makes me seethe to see it.

Comment (EOBGames) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm:263]: We're in a bold new age of oil adulteration

**Comment by MidoriWroth** [comment]
I like your ideas, but the quality oil one is a bit odd to me. I planned on PR'ing my olives again as seen in #66946 once i get enough gbp to do so, so I wonder which you would prefer. To me, it makes more sense if you can get the oil from the olives instead, but in an extremely inefficient reaction like you would do in real life.

File: code/modules/food_and_drinks/recipes/food_mixtures.dm (hunk 1 above)
Comment (VladinXXV) [reply code/modules/food_and_drinks/recipes/food_mixtures.dm:263]: Delightfully devilish, Seymour....

**Comment by ilikesaying** [comment]
~~add universal enzyme to the quality oil reaction for better handwaving~~

**Comment by CRITAWAKETS** [comment]
wait, if quality oil requires quality oil to make

how was the first quality oil made??

**Comment by TemporalOroboros** [comment]
> Yoghurt
I know this isn't quite a this-pr thing, but how should cheese and sour cream be differentiated?

**Comment by TemporalOroboros** [comment]
> wait, if quality oil requires quality oil to make
> 
> how was the first quality oil made??

Maybe magic? 
I mean that literally.
NT could have imported it from the wizards.

**Comment by Bluedino1025** [comment]
Maybe for the oven, you could add stock parts to it that way if you want it to cook faster you need to put in a new capacitor or something with the rped.

**Comment by VladinXXV** [comment]
> Maybe for the oven, you could add stock parts to it that way if you want it to cook faster you need to put in a new capacitor or something with the rped.

I think the oven already uses lasers as a stock part, whether or not it actually does anything I am not sure.

**Comment by Dorsisdwarf** [comment]
I actually think cutting oven times in half is a big mistake. It now means oven times are still annoyingly long if you have everything else in front of you, but you can no longer leave it to pop in the backroom and get something else out of the cabinets

**Comment by gitbirb** [comment]
Hooray! It was so awful trying to make a single PIZZA that barely gave any food for an hour, and still never being able to because it was so difficult to craft (even after practicing the recipes I couldn't do anything in less then like, 30-40 minutes, which is insane for any food, and i just gave up). Hopefully this will make it actually fun to make.

**Pull request merged**